userguide/drawing_classifier/advanced-usage.md

```diff
@@ -22,15 +22,13 @@ provided value of `num_epochs` would be ignored.
 
 To boost the accuracy of the Drawing Classifier models you train, and to help
 those models converge faster, we provide the option of loading in a 
-pretrained model for a warm start. 
+pretrained model for a warm start.
 
 A cold start to training would be when the weights are randomly initialized. 
 A warm start is when the weights in the neural network are loaded from those 
 of an already trained model. This improves the initial values of the weights 
 in the network to something better than random, thereby improving accuracy 
 and also helping the model converge faster.
 
-We have published our pre-trained model 
-[here](https://docs-assets.developer.apple.com/turicreate/). 
-We trained this published model on 1,000 examples each of the first 245 of the
```

Comment:
I think removing the link is a good idea.

This is an advanced section of the user guide. So I think it's good to mention that it was trained on 1,000 example of 245 out 345 classes of the Quick, Draw! dataset. I'm not sure about the phrase "the first 245". Is there some natural ordering of these classes that I'm missing?

Reply:
The natural ordering of these classes on the Quick, Draw! website and in the dataset is lexicographic. I agree that we shouldn't mention "the first 245". The model is trained on 68167 images/class and not 1000 so we should either fix that typo or get rid of it entirely. Maybe something like "This model is trained on millions of drawings from the “Quick,Draw!” dataset"? Sounds general enough that it gives us the opportunity to modify it when we ship a better model down the road, but also gives advanced users a ballpark of how many images the pre-trained model is trained on.